mart/datamodules/modular.py

```diff
@@ -29,6 +29,8 @@ def __init__(
         ims_per_batch=1,
         world_size=1,
         pin_memory=False,
+        # Classification metrics may require the value in config.
+        num_classes=None,
```

Comment:
Why is this necessary if it is never used?

Reply:
The classification metrics in `torchmetrics` requires `num_classes`. I think it's best to specify it in the config of `datamodule` and reference to it in metrics. Otherwise, we have to specify it for every experiments.